src/sotoki/utils/imager.py

```diff
@@ -153,7 +153,7 @@ def get_s3_key_for(self, url: str) -> str:
 
     def get_digest_for(self, url: str) -> str:
         """Unique identifier of that url"""
-        return hashlib.md5(url.encode("UTF-8"), usedforsecurity=False).hexdigest()
+        return hashlib.md5(url.encode("UTF-8")).hexdigest()
```

Comment:
Thank you! You need to re-request the review if you're ready.
Before you do that, can you add a `  # nosec` comment at the end of the line so Codefactor doesn't complain?